demo-repository/exercises/demo2/descr.md

```diff
@@ -1,4 +1,4 @@
-
+[ceci est un lien](http://google.com)
```

Comment:
Good idea to add this test here; but:

* Could you propose a more OCaml-oriented link, and in English? e.g.:
```suggestion
The following example link will open another tab/window: [OCaml](https://ocaml.org/).
```

* BTW did you test what we get for internal links? I mean, `[Internal link](#subsection)` should not generate a link with `target="_blank" etc.` because the href starts with a `#`…

Reply:
Hello @erikmd ,
thanks for your comments. Good idea for a more OCaml-oriented link ! :)

So, you were right, i had to update the code quickly to handle internal links. After few tests, I realize that it was OK but Omd seems to translate weirdly sometimes, for example on the 2 screens below you can see that the "?" is remove from the first id of the title and for the second one, Omd replace the apostrophe by "-39-" and also "é" by "-". 
I don't know if it is caused by HTML which don't allow accent and stuff like this in id.
![test](https://user-images.githubusercontent.com/46816453/118244065-6d363180-b49f-11eb-820e-3ab0ac85c260.png)
![test2](https://user-images.githubusercontent.com/46816453/118244079-6f988b80-b49f-11eb-9007-a7593d0d50ee.png)

Reply:
@LouisAyroles

> Omd seems to translate weirdly sometimes, for example on the 2 screens below you can see that the "?" is remove from the first id of the title and for the second one, Omd replace the apostrophe by "-39-" and also "é" by "-".
I don't know if it is caused by HTML which don't allow accent and stuff like this in id.

Actually I don't see any issue here: it is very common that the Markdown-to-HTML converters generate an `id=` in pure ASCII without any space… so we could say this is not a bug but a feature ;)